webui/react/src/components/ResourcePoolCard.tsx

```diff
@@ -170,15 +171,15 @@ const ResourcePoolCard: React.FC<Props> = ({
             </div>
             <div className={css.default}>
               {showDescriptiveLabel && <span>{descriptiveLabel}</span>}
-              {pool.description && <Icon name="info" showTooltip title={pool.description} />}
-            </div>
-            {!showDescriptiveLabel && (
-              <div className={css.defaultPoolTooltip}>
+              {showDescriptiveLabel && pool.description && (
```

Comment:
fix: I don't see any reason why the description should depend on `showDescriptiveLabel`. descriptiveLabel just refers to the Default/Default Aux/Default Compute

Reply:
noting the follow up PR will refactor this area complete